### ISSUE TITLE
bad Javascript: "Uncaught TypeError: Cannot read property 'innerHTML' of null"

### ISSUE THREAD

**Issue by The-Alchemist** [unclebob/fitnesse#1048]
## Offending Javascript

```
var existingContent = document.getElementById("test-system-slim:fitnesse.slim.SlimService").innerHTML;
```

I actually get this four times in one page.

## Specs
 * $ java -version
java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
* Fitnesse 20160618
* Chrome 58.0.3029.110 (64-bit)

Unfortunately, it's part of a much larger project and I can't post it. :(  I'm just wondering if anyone has seen this.

**Comment by amolenaar** [comment]
This happens when you do a suite run,  right? It's part of the code that updates the top part of the page.

**Comment by The-Alchemist** [comment]
@amolenaar : Yup, you got it!  I'm guessing you've seen it already?  Any tips?  I don't mind submitting a PR if someone points me in the right direction.

**Comment by amolenaar** [comment]
It would be nice to modernize the Javascript code on the suite page. I suppose `SuiteHtmlFormatter.java` is a good starting point.
